packages/ffe-tables-react/src/SortableTable/SortableTable.js

```diff
@@ -37,8 +32,18 @@ class SortableTable extends Component {
                           prevState.descending,
                       )
                     : nextProps.data,
-            }));
+                props: nextProps,
```

Comment:
This is a pragmatic way of doing things, but I think we can do even better. Saving all the previous props in state seems like an anti-pattern we can avoid.

Perhaps we can memoize the `sortData` method, and call it on every render? This way we can omit keeping the sorted table data in state as well! 

You can read more about this approach in the ["You might not need derived state"](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#what-about-memoization) blog post by the React team.